app/views/shared/vacancy/_jobseeker_view.html.slim

```diff
@@ -37,6 +37,7 @@
                                                  link_id: "#{@saved_job.present? ? 'saved' : 'save'}-job-listing",
                                                  link_method: @saved_job.present? ? :delete : :get,
                                                  link_path: @saved_job.present? ? jobseekers_saved_job_path(@vacancy.id, @saved_job) : new_jobseekers_saved_job_path(@vacancy.id),
+                                                 params: { click_event: "vacancy_save_to_account_clicked" },
```

Comment:
Would @stevenleggdfe prefer this to have a distinct name from when it was triggered in the wrong place, so that it's easy to disambiguate?

Reply:
@david-mears-dfe he wouldn't, because he's just going to delete all the old events once this PR is merged :-)